File: kotlin/src/HelloLoop.kt
fun main() {
    var worldArray = arrayOf("a", "b", "c")

    for (world in worldArray) {
        println("world is $world!")
    }

    var worldMap = mapOf("a" to "aaa", "b" to "bbb", "c" to "ccc")

    for ((key, value) in worldMap) {
        println("world $key is $value!")
    }
}

File: golang/src/hello_loop.go
package main

import "fmt"

func main() {
	worldArray := []string{"a", "b", "c"}
	for _, world := range worldArray {
		fmt.Printf("world is %s!\n", world)
	}

	worldMap := map[string]string{"a": "aaa", "b": "bbb", "c": "ccc"}

	for key, value := range worldMap {
		fmt.Printf("world %s is %s!\n", key, value)
	}
}
